src/tools/generateOptionsSchema.ts

```diff
@@ -379,9 +379,9 @@ export function GenerateOptionsSchema() {
     // Add the additional options we do want
     unitTestDebuggingOptions['type'] = {
         type: 'string',
-        enum: ['coreclr', 'clr'],
+        enum: ['coreclr', 'clr', 'monovsdbg'],
```

Comment:
>, 'monovsdbg' [](http://example.com/codeflow?start=31&length=13)

I think we shouldn't advertise this as it will probably never work -- mono requires a bunch of other configuration that the project system does.